include/cone_detector/cone_detector.h

```diff
@@ -120,9 +127,17 @@ class ConeDetector {
 	std::string ur_annotation_;
 	cv::Scalar ur_color_;
 
+	// Process CalibrateConeDetection service call.
+	ros::ServiceServer calibrateConeDetectionService;
+
+	// Handle annotation service.
 	bool annotateCb(victoria_perception::AnnotateDetectorImage::Request &request,
 			victoria_perception::AnnotateDetectorImage::Response &response);
 
+	// Handle calibrate cone detection service.
+	bool calibrateConeDetectionCb(victoria_perception::CalibrateConeDetection::Request &request,
```

Comment:
Doxygen will have to wait for a future pass. Fixed const parameters where allowed. Some of those provided by the ROS framework don't allow consts, such as dynamic configuration callbacks.